lib/web_ui/lib/src/engine/skwasm/skwasm_impl/canvas.dart

```diff
@@ -32,28 +32,41 @@ class SkwasmCanvas implements SceneCanvas {
     final paintHandle = (paint as SkwasmPaint).toRawPaint();
     if (bounds != null) {
       withStackScope((StackScope s) {
-        canvasSaveLayer(_handle, s.convertRectToNative(bounds), paintHandle, nullptr);
+        canvasSaveLayer(_handle, s.convertRectToNative(bounds), paintHandle, nullptr,
+                        ui.TileMode.clamp.index);
       });
     } else {
-      canvasSaveLayer(_handle, nullptr, paintHandle, nullptr);
+      canvasSaveLayer(_handle, nullptr, paintHandle, nullptr, ui.TileMode.clamp.index);
     }
     paintDispose(paintHandle);
   }
 
   @override
   void saveLayerWithFilter(ui.Rect? bounds, ui.Paint paint, ui.ImageFilter imageFilter) {
+    // There are 2 ImageFilter objects applied here. The filter in the paint
+    // object is applied to the contents and its default tile mode is decal
+    // (automatically applied by toSkPaint).
+    // The filter supplied as an argument to this function [nativeFilter] will
+    // be applied to the backdrop and its default tile mode will be mirror.
+    // We also pass in the blur tile mode as an argument to saveLayer because
+    // that operation will not adopt the tile mode from the backdrop filter
+    // and instead needs it supplied to the saveLayer call itself as a
+    // separate argument.
     final SkwasmImageFilter nativeFilter = SkwasmImageFilter.fromUiFilter(imageFilter);
-    final paintHandle = (paint as SkwasmPaint).toRawPaint();
+    final ui.TileMode? backdropTileMode = nativeFilter.backdropTileMode;
+    final paintHandle = (paint as SkwasmPaint).toRawPaint(/*ui.TileMode.decal*/);
```

Comment:
I don't really love this little inline comment here. Either we actually want the argument to be visible, and we shouldn't provide a default value, or we should just nix this comment and allow the default argument to be implicit and not visible in the code. (Also, the comment should probably be like `/*defaultBlurTileMode: ui.TileMode.decal*/` since it is a named argument, but I we should just remove it).

Reply:
The comment above this talks about how there are 2 different filters involved in this operation and one uses mirror and the other uses decal as the defaults. If this comment isn't there then you can see where "mirror" is used, but there is no indication of where "decal" is used because it is implicit. I tried just specifying it manually (without the comments), but the analyzer rejected the idea of manually supplying the value that is the default (why?). So, I put it in comments to satisfy the analyzer while still be indicative of the "2 different defaults" I had described earlier.

Why won't the analyzer just let me say the default without complaining?

Reply:
If I provide no default in the call arguments then every draw command in this file has to supply decal even though it is the default.